### PR TITLE
Add requirements file generation for quantizer.

### PR DESCRIPTION
**Pull request by nignatovsky**

### CODE REVIEW

File: ci/update-requirements.py [hunk 1]
```diff
@@ -76,6 +76,10 @@ def pc(target, *sources):
         'demos/python_demos/requirements.txt', openvino_dir / 'python/requirements.txt')
     pc('ci/requirements-downloader.txt',
         'tools/downloader/requirements.in')
+    pc('ci/requirements-quantization.txt',
+        'tools/accuracy_checker/setup.py',
```
Comment (IRDonch) [inline ci/update-requirements.py:80]: Please use `requirements-core.in`, and `requirements.in` instead (see the call that generates `requirements-ac.txt`).

File: ci/update-requirements.py [hunk 2]
```diff
@@ -76,6 +76,10 @@ def pc(target, *sources):
...
+        openvino_dir / 'deployment_tools/tools/post_training_optimization_toolkit/setup.py',
+        openvino_dir / 'deployment_tools/model_optimizer/requirements.txt')
```
Comment (IRDonch) [inline ci/update-requirements.py:82]: I suspect that POT doesn't actually need all of the frameworks that MO supports. I suggest that you try to use one of MO's `requirements_<framework>.txt` files instead (preferably the one with the least amount of dependencies in it) and see if POT still works.

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: ci/update-requirements.py (hunk 1 above)
Comment (nignatovsky) [reply ci/update-requirements.py]: Done. (Downloader Job [logs](http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/466/console))

File: ci/update-requirements.py (hunk 2 above)
Comment (nignatovsky) [reply ci/update-requirements.py]: Done. (Works with kaldi requirements; [logs](http://213.221.44.203/job/omz/job/precommit/job/downloader_ubuntu18/466/console))

**Review by IRDonch — APPROVED** [review]

**Comment by IRDonch** [comment]
Please also update `.gitattributes` so that changes to `requirements-quantization.txt` cause the downloader job to run.

**Comment by IRDonch** [comment]
Jenkins: test this please

**Pull request merged**